priam/src/main/java/com/netflix/priam/cluster/management/TaskException.java

```diff
@@ -0,0 +1,15 @@
+package com.netflix.priam.cluster.management;
+
+/**
+ * Created by aagrawal on 2/28/18.
+ */
+public class TaskException extends Exception{
```

Comment:
What extra functionality is this giving apart from regular Exception? Are you looking for any specific TaskExceptions to act differently? I dont see that in code.

Reply:
+1 Deleting. Since we already handle the counter increment in IClusterManagement. This does not make sense.